routes/postevents/postevents.js

```diff
@@ -0,0 +1,35 @@
+// endpoints related to postevent
+const express = require('express');
+const pool = require('../../db');
+
+const posteventRouter = express();
+
+posteventRouter.use(express.json());
+
+// Get Post Event
+posteventRouter.get('/:eventId', async (req, res) => {
+  try {
+    const getPostById = await pool.query('SELECT * FROM postevent WHERE event_id = $1;', [
+      req.params.eventId,
+    ]);
+    res.status(200).json(getPostById.rows[0] || {});
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+// Create Post Event
+posteventRouter.post('/create', async (req, res) => {
+  try {
+    const { eventId, description } = req.body;
+    const createPostEvent = await pool.query(
```

Comment:
add query that checks if postevent already exists for event, if so then don't create one